blocks/media/media.css

```diff
@@ -0,0 +1,230 @@
+    transition: opacity 1s ease;
+}
+
+.block.media .dots-container .dot {
+    height: 0.9rem;
+    width: 0.9rem;
+    margin: 0 0.3rem;
+    padding: 0;
+    cursor: pointer;
+    background: var(--background-color-dark-grey);
+    border-radius: 3rem;
+    border: 0;
+}
+
+.block.media .dots-container .dot.active{
+    background: var(--link-color-orange);
+}
+
+.block.media .dots-container {
+    text-align: center;
+    margin-top: 1.3rem;
+}
+
+.block.media .discovermore {
+    text-align: center;
+    padding-top: 1.5rem;
+}
+
+.block.media .card-info .text-content .card-title a {
+    color: var(--title-color-card);
+    font-weight: 500;
+    font-size: 1.25rem;
+    word-break: break-word;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: -webkit-box;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
+    text-decoration: none;
+}
+
+.block.media .card-info .text-content .card-meta {
+    display: flex;
+    flex-wrap: wrap;
+    margin-bottom: var(--space-8px);
+}
+
+.block.media .card-info .text-content .card-meta abbr {
+    font-size: var(--space-11px);
+    color: var(--card-post-color);
+    padding-right: var(--space-5px);
+    border-right: var(--space-1px) solid var(--card-post-boarder);
+    line-height: var(--space-10px);
+    margin: 0 var(--space-4px) var(--space-8px) 0;
+}
+
+.block.media  .card-info .text-content .card-meta abbr:last-child {
+    border: 0;
+}
+
+.block.media .card-info .text-content .social-link {
+    list-style: none;
+    display: flex;
+    margin-top: var(--space-8px);
+    padding-top: 1rem;
+    padding-left: 0;
+}
+
+.block.media .card-info .text-content .social-link a {
+    border-radius: var(--space-4px);
+    border: var(--space-1px) solid var(--link-color-gray);;
+    padding: var(--space-5px);
+    width: 2.125rem;
+    height: 2.125rem;
+    text-align: center;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    cursor: pointer;
+}
+
+.block.media .text-content .social-link a .icon {
+    color: var(--link-color-gray);
+    font-size: 1.25rem;
+}
+
+.block.media .text-content .social-link .fa {
+    display: inline-block;
+    font: normal normal normal var(--space-14px) / 1 font-awesome, sans-serif;
+    font-size: inherit;
+    text-rendering: auto;
+}
+
+.block.media .text-content .social-link .fa-share::before {
+    content: '\f064';
+}
+
+.block.media .card-info .picture-wrapper {
+    position: relative;
+    margin-bottom: 2.188rem;
+    cursor: pointer;
+    border-radius: var(--space-8px);
+}
+
+.block.media .card-info .picture-wrapper::after {
+    content: "";
+    display: block;
+    width: 3.625rem;
+    height: 3.625rem;
+    background-size: 100%;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    margin: -1.813rem 0 0 -1.813rem;
+    z-index: 10;
+}
+
+.block.media .card-info.videos-card .picture-wrapper::after {
+    background-image: url("../../icons/play-button.svg");
```

Comment:
is it possible to use CreateIcon method from aem.js ?

Reply:
I tried but didn't work , as this is sudo element to create a overlay of play button on video thumbnail